sources/packages/web/src/components/institutions/modals/AddRestrictionModal.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <v-form ref="addRestrictionForm">
+    <modal-dialog-base title="Add new restriction" :show-dialog="showDialog">
+      <template #content>
+        <error-summary :errors="addRestrictionForm.errors" />
+        <v-select
+          item-value="id"
+          item-title="description"
+          class="my-4"
+          label="Reason"
+          density="compact"
+          :items="reasons"
+          v-model="formModel.restrictionId"
+          variant="outlined"
+          :rules="[(v) => !!v || 'Reason is required.']"
```

Comment:
We have a composable `checkNullOrEmptyRule` but we use this way in many places. I would suggest composable.

Reply:
I will change it.